coredb-cli/src/commands/create.rs

```diff
@@ -17,7 +17,7 @@ fn generate_yaml(resource_type: ResourceType, name: String) -> String {
     match resource_type {
         ResourceType::Db | ResourceType::Dbs => {
             let json_value = json!({
-                "apiVersion": "kube.rs/v1",
```

Comment:
Thanks! @sjmiller609 what are your thoughts on starting with `coredb.io/v1alpha1`? We will likely have new API versions in the future.

Reply:
ok done